### DISCUSSION TITLE
How to remove categories prefix from URL?

### DISCUSSION THREAD

**Discussion by Bikersguide** [chipzoller/hugo-clarity#444] [General]
Hi everyone!

How to remove categories prefix from URL?

I have tried with

```
[permalink]
   categories = "/:slug/"
```

in config.toml

ERROR 404!

Is there a way to remove the categories prefix from the URL?

Thanks!

**Comment by onweru** [comment]
<img width="591" alt="SCR-20230714-jwmq" src="https://github.com/chipzoller/hugo-clarity/assets/16350351/8bd646f3-e6cc-477b-a661-74c82ecb172c">

Check [this documentation page](https://gohugo.io/content-management/urls/#permalinks) for more info

**Reply by Bikersguide** [reply]
Hi onweru!

Thank you for your quick response.

The link generated to a category in the sidebar is incorrect.
After changing the config.toml the categories prefix is present in the URL that is generated in the sidebar and I get the ERROR 404.

How can I change this?
